static/app/views/dashboardsV2/dashboard.tsx

```diff
@@ -568,18 +570,21 @@ class Dashboard extends Component<Props, State> {
         rowHeight={ROW_HEIGHT}
         margin={WIDGET_MARGINS}
         draggableHandle={`.${DRAG_HANDLE_CLASS}`}
+        draggableCancel={`.${DRAG_RESIZE_CLASS}`}
```

Comment:
This is a bit better than trying to override all the styles on `react-resizable-handle`